jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/FileManager.java

```diff
@@ -1959,9 +1959,9 @@ public static void generateCloudTrailFiles(Map<String,List<CloudTrailVH>> cloudT
 		String keys;
 		fieldNames = "trail.Name`trail.S3BucketName`trail.IncludeGlobalServiceEvents"
 				+ "`trail.IsMultiRegionTrail`trail.HomeRegion`trail.TrailARN`"
-				+ "trail.LogFileValidationEnabled`trail.HasCustomEventSelectors`logginEnabled`trail.KmsKeyId";
+				+ "trail.LogFileValidationEnabled`trail.HasCustomEventSelectors`logginEnabled`trail.KmsKeyId`trail.cloudWatchLogsLogGroupArn`latestCloudWatchLogsDeliveryTime";
 		keys = "discoverydate`accountid`accountname`region`name`s3bucketname`includeglobalserviceevents"
-				+ "`ismultiregiontrail`homeregion`trailarn`logfilevalidationenabled`hascustomeventselectors`islogging`kmskeyid";
+				+ "`ismultiregiontrail`homeregion`trailarn`logfilevalidationenabled`hascustomeventselectors`islogging`kmskeyid`cloudWatchLogsLogGroupArn`latestCloudWatchLogsDeliveryTime";
```

Comment:
I understand the intention behind reusing constants,but I am simply following the previous pattern."also
if we use constants then there will be error in this line
String[] keysList = keys.split("["+DELIMITER+"]+");
where DELIMITER="`"

Reply:
we should produce a good code, not just follow what has been here before.
Could you point me to exact location where this code is, and what error you expect if you will be using constant instead of string?